content/messaging/agent-chat-api/index.mdx

```diff
@@ -826,25 +827,25 @@ Here's the list of all system messages you might come across:
   "customer_last_event_created_at": "2017-10-12T15:19:21.010200Z",
   "created_at": "2017-10-11T15:19:21.010200Z",
   "present": true, // optional, applies only to customer located in chat object
-  "events_seen_up_to": 1473433500 // optional, applies only to customer located in chat object
+  "events_seen_up_to": "2017-10-12T15:19:21.010200Z"
 }
 ```
 
 </CodeResponse>
 
-| Field                            | Req./Opt. |                                      Note |
+| Field                            | Req./Opt. |                                     Notes |
 | -------------------------------- | :-------: | ----------------------------------------: |
-| `agent_last_event_created_at`    | optional  |                                         - |
-| `avatar`                         | optional  |                                         - |
-| `customer_last_event_created_at` | optional  |                                         - |
-| `created_at`                     | optional  |                                         - |
-| `email`                          | optional  |                                         - |
+| `agent_last_event_created_at`    | optional  |                                           |
+| `avatar`                         | optional  |                                           |
+| `customer_last_event_created_at` | optional  |                                           |
+| `created_at`                     | optional  |                                           |
+| `email`                          | optional  |                                           |
 | `fields`                         | optional  | Is not present when the chat is archived. |
-| `name`                           | optional  |                                         - |
-| `last_seen_timestamp`            | optional  |                                         - |
-| `last_visit`                     | optional  |                                         - |
-| `present`                        | optional  |                                         - |
-| `statistics`                     | optional  |                                         - |
+| `name`                           | optional  |                                           |
+| `events_seen_up_to`              |           | RFC 3339 datetime string                  |
```

Comment:
Is it also string in RFC format? Required?

Reply:
Yes and yes :)

Reply:
Or actually it's optional (as other fields here)